src/static/assets/css/markdown.css

```diff
@@ -0,0 +1,695 @@
+@font-face {
```

Comment:
From where did you get this CSS? What is its license? Can we just link to the external file, hosted somewhere else?If not, add a comment explaining where we got it.